metron-interface/pom.xml

```diff
@@ -48,12 +48,6 @@
         <module>metron-rest-client</module>
     </modules>
     <dependencies>
-        <dependency>
```

Comment:
Apparently from `metron-integration-test`, because it has its stuff in source, so pulling that in pulls the JUnit dep. I can make it explicit here, if you want?

Reply:
I think we can consider how best to deal with this in a clean up.